frontend/routes/dashboard/dashboard.component.tsx

```diff
@@ -115,7 +112,7 @@ export class Dashboard extends React.PureComponent<IProps, any> {
 				<Sidebar>
 					<UserInfo
 						{...currentUser}
-						loading={!isInitialized && (isPending || isAvatarPending)}
+						loading={false}
```

Comment:
You can remove this prop ;)

Reply:
That was for debugging that I forgot to revert.